app/helpers/url_helper.rb

```diff
@@ -14,4 +14,8 @@ def github_tag_link_to(app, git_ref)
   def github_compare_to_default(app, deploy)
     "#{github_repo_url(app)}/compare/#{deploy.version}...#{app.default_branch}"
   end
+
+  def argo_app_link_to(app, environment)
+    link_to(environment, "https://argo.eks.#{environment}.govuk.digital/applications/#{app.downcase}", target: "_blank", rel: "noopener", class: "govuk-link")
```

Comment:
The link won't work if the name doesn't match. We may need to store it in the DB (e.g. `k8s_name`). Then we won't need the k8s label.